### ISSUE TITLE
4.0.5: Hotkey L to 'Hide/show the folder pane' does not work yet

### ISSUE THREAD

**Issue by ecogit** [opto/nostalgy-xpi#228]
Since upgrading from 102 to 115 has this shortcut stopped functioning. 

Nostalgy++ 4.0.5
Thunderbird 115.3.1
Ubuntu 23.04

**Comment by opto** [comment]
yes, the folderpane has been rewritten in core and the old commands don't exist anymore.

By the way, there is a native TB shortcut doing the same.

**Comment by ecogit** [comment]
Oh, what a pity those hotkeys are gone, they added to the greatness of Nostalgy. 

Yes, the lengthy shortcut sequence is `alt + v + l + f`. It seems to work on both, Windows and Linux. But it isn't listed on Thunderbird's [Keyboard shortcuts](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-thunderbird) documentation, so probably most users will fall back on hovering the mouse.

Also, there exists the add-on [tbkeys-lite](https://addons.thunderbird.net/en-US/thunderbird/addon/tbkeys-lite), which allows to customize Thunderbird's keybindings, but by default it doesn't come with a configuration which toggles the folder pane.

In case one day the situation changes and it became a low-hanging fruit for Nostalgy to bring these hotkeys back, that would be a plus. 

For the meantime, I guess we best close this issue.

**Comment by ecogit** [comment]
Thank you @opto :)

I can confirm that it works with [manage_emails_nostalgy_pp_4.1.0-tb.xpi](https://github.com/opto/nostalgy-xpi/releases/download/4.1.0/manage_emails_nostalgy_pp_4.1.0-tb.xpi). 

Besides of Linux, I also tested on macOS and Win11.
